resources/config_ldap.rb

```diff
@@ -0,0 +1,64 @@
+#
+# Cookbook Name:: grafana
+# Resource:: config_alerting
+#
+# Copyright 2019, Sous Chefs
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Configures the installed grafana instance's ldap settings
+# See https://raw.githubusercontent.com/grafana/grafana/master/conf/ldap.toml
+
+property  :instance_name,                 String, name_property: true
+property  :conf_directory,                String, default: '/etc/grafana'
+property  :config_file,                   String, default: lazy { ::File.join(conf_directory, 'ldap.toml') }
+property  :log_filters,                   String
+property  :servers_attributes_name,       String, default: 'givenName'
```

Comment:
Could you make `server_attributes` a hash for name, surname, etc. as keys?

Reply:
Could do but so far everything has been an explicit property, feels like it would be an oddball in this cookbook?

Reply:
Yeah, consistent feel seems better